### PR TITLE
Create blank.yml.2

### PR DESCRIPTION
**Pull request by Pokipoki057**
# This is a basic workflow to help you get started with Actions

name: CI

# Controls when the action will run. 
on:
  # Triggers the workflow on push or pull request events but only for the $default-branch branch
  push:
    branches: [ $default-branch ]
  pull_request:
    branches: [ $default-branch ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - uses: actions/checkout@v2

      # Runs a single command using the runners shell
      - name: Run a one-line script
        run: echo Hello, world!

      # Runs a set of commands using the runners shell
      - name: Run a multi-line script
        run: |
          echo Add other actions to build,
          echo test, and deploy your project.

### CODE REVIEW

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    # Runs a single command using the runners shell
+    - name: Run a one-line script
+      run: echo Hello, world!
+
+    # Runs a set of commands using the runners shell
+    - name: Run a multi-line script
+      run: |
+        echo Add other actions to build,
+        echo test, and deploy your project.
```
Comment (johnperez416) [inline .github/workflows/blank.yml:33]: <details>
<summary>Details</summary>



</details>

File: .github/workflows/blank.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
```
Comment (natasha1007) [inline .github/workflows/blank.yml:19]: .github/workflows/blank.yml

**Review by natasha1007 — COMMENTED** [review]
.github/workflows/blank.yml

**Comment by stonehead8111** [comment]
cd

**Comment by stonehead8111** [comment]
e

**Pull request closed without merging**